tests/strategies/test_ddp_hydra_support.py

```diff
@@ -0,0 +1,157 @@
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import pytest
+
+from pytorch_lightning.utilities.imports import _HYDRA_AVAILABLE
+from tests.helpers.runif import RunIf
+
+if _HYDRA_AVAILABLE:
+    from omegaconf import OmegaConf
+
+
+# fixture to run in a clean temporary directory
+@pytest.fixture()
+def cleandir():
```

Comment:
You could use tmpdir provided by pytest there.

Reply:
oh cool, I wasn't aware of that, I'll update.